packages/@sanity/desk-tool/src/pane/History.js

```diff
@@ -0,0 +1,98 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import scroll from 'scroll'
+import CloseIcon from 'part:@sanity/base/close-icon'
+import Button from 'part:@sanity/components/buttons/default'
+import Snackbar from 'part:@sanity/components/snackbar/default'
+import Spinner from 'part:@sanity/components/loading/spinner'
+import HistoryItem from './HistoryItem'
+
+import styles from './styles/History.css'
+
+const maybe = (val, fn) => val && fn(val)
+
+export default class History extends React.PureComponent {
+  static propTypes = {
+    onClose: PropTypes.func,
+    documentId: PropTypes.string,
+    onItemSelect: PropTypes.func,
+    lastEdited: PropTypes.object,
+    errorMessage: PropTypes.string,
+    selectedEvent: PropTypes.object
+  }
+
+  state = {
+    headerShadowOpacity: 0
+  }
+  _listElement = React.createRef()
+
+  componentDidMount() {
+    if (this._listElement.current && this._listElement.current) {
```

Comment:
This seems wrong. Should this be `this._listElement && this._listElement.current`?